packages/f/freefilesync/pspec_x86_64.xml

```diff
@@ -21,125 +22,32 @@
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/french.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/german.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/greek.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/hebrew.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/hindi.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/hungarian.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/italian.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/japanese.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/korean.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/lithuanian.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/norwegian.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/polish.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/portuguese.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/portuguese_br.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/romanian.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/russian.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/slovak.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/slovenian.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/spanish.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/swedish.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/turkish.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Languages/ukrainian.lng</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/Resources.zip</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/ding.wav</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/gong.wav</Path>
-            <Path fileType="data">/usr/share/FreeFileSync/harp.wav</Path>
             <Path fileType="data">/usr/share/applications/FreeFileSync.desktop</Path>
             <Path fileType="data">/usr/share/applications/RealTimeSync.desktop</Path>
-            <Path fileType="doc">/usr/share/doc/FreeFileSync/CHANGELOG.gz</Path>
+            <Path fileType="data">/usr/share/freefilesync/Gtk2Styles.rc</Path>
+            <Path fileType="data">/usr/share/freefilesync/Gtk3Styles.css</Path>
+            <Path fileType="data">/usr/share/freefilesync/Gtk3Styles.old.css</Path>
+            <Path fileType="data">/usr/share/freefilesync/Icons.zip</Path>
+            <Path fileType="data">/usr/share/freefilesync/Languages.zip</Path>
```

Comment:
Does Debian do anything with this? I just want to make sure they don't need to be unzipped in order to be used.

Reply:
It's read by the application so it doesn't need to be unzipped.

Reply:
Sweet